foundry.toml

```diff
@@ -95,6 +95,9 @@ scroll = "${ETH_NODE_URI_SCROLL}"
 # testnets
 localanvil = "${ETH_NODE_URI_LOCALANVIL}"
 
+[lint]
+lint_on_build=false
```

Comment:
this is helpful when using newest foundry version as they changed it to default=true